src/node/rpc/frontend.h

```diff
@@ -434,6 +434,26 @@ namespace ccf
           return jsonrpc::success(out);
         };
 
+      auto who_am_i = [this](const RequestArgs& args) {
+        return jsonrpc::success(WhoAmI::Out{args.caller_id});
```

Comment:
On the nodes frontend, `args.caller_id` will be set to `INVALID_ID`. We should return an error if `certs == nullptr` instead.